README.md

```diff
@@ -34,25 +36,13 @@ Once the project is built from the template, there are manual settings to be con
 ### Package Deployment
 See [releasing scipp](https://scipp.github.io/reference/developer/releasing-scipp.html#updating-an-expired-anaconda-token) for more information about deployment.
 
-1. Conda
+#### Conda
 Go to Settings > Secrets > Actions > Organization secrets.
 There is `scipp` organization-wide anaconda key, `ANACONDATOKEN`. But it should be enabled per repository.
 
-2. Pypi
-Go to Settings > Secrets > Actions > Repository secrets.
-`PYPI_TOKEN` should be configured per repository.
-You may need to create a token under your private account on [pypi.org](https://pypi.org/).
-If it is the first time of deployment, there are 2 ways to upload the package.
-
-1. Manually build and upload the wheel.
-  See [upload the distribution archives](https://packaging.python.org/en/latest/tutorials/packaging-projects/#uploading-the-distribution-archives) page for more details.
-  You may need to use your personal username/password to upload the package.
-
-2. Create a token that has access to all-projects under your account.
-  In order to create a new project, the token used for the deployment needs access to all projects under your account.
-  It is because you can not select a project that does not exist in `pypi` yet.
+#### PyPI
 
-After the first deployment, a new token that has access to the project can be created in [pypi.org](https://pypi.org/) and configured in the repository.
+Create a new project by [adding a trusted publisher](https://docs.pypi.org/trusted-publishers/creating-a-project-through-oidc/).
```

Comment:
This also requires setting up the release environment on GH

Reply:
Right, do we have recommendation on the settings for the env?

Reply:
> that doesn't work so well because the action releases from some other branch.

You mean it is not `main`, because we release from a tag?